src/gl.rs

```diff
@@ -1112,6 +1112,24 @@ pub fn get_program_iv(program: GLuint, pname: GLenum) -> GLint {
 }
 
 #[inline]
+pub fn get_vertex_attrib_iv(index: GLuint, pname: GLenum) -> GLint {
+    unsafe {
+        let mut result: GLint = 0 as GLint;
+        ffi::GetVertexAttribiv(index, pname, &mut result);
+        return result;
+    }
+}
+
+#[inline]
+pub fn get_vertex_attrib_fv(index: GLuint, pname: GLenum) -> Vec<GLfloat> {
+    unsafe {
+        let mut result = vec![0 as GLfloat; 4];
```

Comment:
I don't see anything wrong with the way I call ffi and cargo points to http://doc.servo.org/src/gleam/home/servo/buildbot/slave/doc/build/target/debug/build/gleam-e640bc53dabe9ccf/out/gl_bindings.rs.html#4328-4345 which doesn't seem very helpful. Any ideas @emilio ?